crates/egui/src/input_state.rs

```diff
@@ -54,6 +54,14 @@ pub struct InputState {
     /// Position and size of the egui area.
     pub screen_rect: Rect,
 
+    /// Viewport inner position and size, only the drowable area
+    /// unit = physical pixels
+    pub inner_rect: Rect,
+
+    /// Viewport outer position and size, drowable area + decorations
+    /// unit = physical pixels
```

Comment:
Why use physical pixels for these? We use logical points for everything else in egui!

Have you tested this PR with `pixels_per_point != 1`? You can use `egui::gui_zoom::zoom_with_keyboard_shortcuts` in an application to zoom in/out with cmd +/-

Reply:
Because this is for the native window!
Is not breaking any thing when you zoom, because is not used anywhere!

But i can make it to be logical pixels, but only for `InputState`!
In `RawInput` will be physical always!

Reply:
But when you set the size or position of the window with a `ViewportCommand`
Should not be physical pixels?

Reply:
`inner_rect` and `outer_rect` was made to position a new viewport, but winit doesn't work well with that!
So can be removed if you want!